.buildkite/scripts/steps/artifacts/docker_image.sh

```diff
@@ -98,14 +98,27 @@ tar -xf "kibana-$BASE_VERSION-cdn-assets.tar.gz" -C "$CDN_ASSETS_FOLDER" --strip
 gsutil -m cp -r "$CDN_ASSETS_FOLDER/*" "gs://$GCS_SA_CDN_QA_BUCKET/$GIT_ABBREV_COMMIT"
 gcloud auth revoke "$GCS_SA_CDN_QA_EMAIL"
 
+echo "--- Validate CDN assets"
+CDN_DESTINATION="https://kibana-cdn.gcp.qa.cld.elstc.co"
+cd $CDN_ASSETS_FOLDER
+shopt -s globstar
+for CDN_ASSET in **/*
```

Comment:
It may be worth running this in parallel to save minutes over a bunch of builds, but don't think it's strictly necessary.

Reply:
I pushed a few commits - https://buildkite.com/elastic/kibana-artifacts-container-image/builds/10188#018dcd0a-6584-4c3d-8103-62e114156106

Reply:
Thanks